MyApplication/app/src/main/java/com/jonathanlieblich/myapplication/SecondDualJobService.java

```diff
@@ -0,0 +1,52 @@
+package com.jonathanlieblich.myapplication;
+
+import android.app.job.JobParameters;
+import android.app.job.JobService;
+import android.os.AsyncTask;
+
+/**
+ * Created by jonlieblich on 11/29/16.
+ */
+
+public class SecondDualJobService extends JobService {
+    AsyncTask<String, Void, String> mTask;
+    PlugChangeListener mListener;
+
+    @Override
+    public boolean onStartJob(final JobParameters jobParameters) {
+        mListener = ServiceListenerHelper.getInstance().getPlug();
+
+        mTask = new AsyncTask<String, Void, String>() {
+            @Override
+            protected String doInBackground(String... strings) {
+                if(strings[0].equals("Unplugged")) {
```

Comment:
Your app crashes here with a `NullPointerException` because `strings` is null. This is because `strings` is populated with the extra you put onto `jobParameters` but you never actually added any extras.